### ISSUE TITLE
Bug: Cannot find module '../screen'

### ISSUE THREAD

**Issue by MicBosi** [Automattic/wp-desktop#219]
A user reported the following error while starting WP desktop app for Mac ("platform":"darwin","release":"15.6.0","version":"1.4.0","build":"release")

```
Error: Cannot find module '../screen'
at Module._resolveFilename (module.js:438:15)
at Function.Module._resolveFilename
(/Applications/WordPress.com.app/Contents/Resources/electron.asar/common/reset-search-paths.js:47:12)
at Function.Module._load (module.js:386:25)
at Module.require (module.js:466:17)
at require (internal/module.js:20:19)
at Object.defineProperties.screen.get [as screen]
(/Applications/WordPress.com.app/Contents/Resources/electron.asar/browser/api/exports/electron.js:83:14)
at setDimensions
(/Applications/WordPress.com.app/Contents/Resources/app/desktop.js:30316:26)
at openWindow
(/Applications/WordPress.com.app/Contents/Resources/app/desktop.js:30332:31)
at Object.module.exports.openAbout
(/Applications/WordPress.com.app/Contents/Resources/app/desktop.js:30354:4)
at menuItems.splice.click
(/Applications/WordPress.com.app/Contents/Resources/app/desktop.js:148050:20)

System info:
{"platform":"darwin","release":"15.6.0","version":"1.4.0","build":"release"}
```

**Comment by chrisfromthelc** [comment]
I've also had a user report the same error message.

**Comment by sendhil** [comment]
I have a fix for this deployed in 1.5.0-beta2. Long story short something in Electron 1.1.1 is causing the require('electron').screen to fail and until I have more time to investigate I rolled back to 1.1.0.
